lmcache/v1/server/__main__.py

```diff
@@ -117,6 +117,17 @@ def handle_client(self, client_socket):
                                 torch.Size((0, 0, 0, 0)),
                             ).serialize()
                         )
+                    case Constants.CLIENT_HEALTH:
+                        client_socket.sendall(
```

Comment:
@zerofishnoodles 
Is there any other type besides SERVER_SUCCESS? Maybe something like `OUT_OF_MEMORY`, `OVER_LOADED` or others?

Reply:
Make sense